hw/ip/entropy_src/dv/cov/entropy_src_extht_exclusions.vRefine

```diff
@@ -28,6 +28,7 @@
     <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;entropy_src_xht_i.continuous_test&quot;" entityType="toggle" excTime="1666055353" name="exclude" reviewer="unknown" user="3" vscope="default"></rule>
     <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;entropy_src_xht_i.test_fail_hi_pulse&quot;" entityType="toggle" excTime="1666055358" name="exclude" reviewer="unknown" user="3" vscope="default"></rule>
     <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;entropy_src_xht_i.test_fail_lo_pulse&quot;" entityType="toggle" excTime="1666055364" name="exclude" reviewer="unknown" user="3" vscope="default"></rule>
+    <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;entropy_src_xht_o.active&quot;" entityType="toggle" excTime="1668165625" name="exclude" reviewer="unknown" user="3" vscope="default"></rule>
```

Comment:
Good catch.

Reply:
I looked at the RTL and it seems like this can toggle. Are we skipping the entire external health test interface altogether from testing?

Reply:
Aah, I just noticed the comment on line 9. I suppose we will delete this vRefine file when we test it in V3?

Reply:
We have the agent in place for it, but we waiving this in M2.  It's not much more work, but it's still one more thing.

Reply:
I am apprehensive of temporarily adding exclusions just to artificially inflate the numbers for an intermediate milestone (because the DV ownership is very fluid). I would instead recommend seeking a waiver in the V2 review meeting for the <90% toggle coverage.

Reply:
Well, this toggle coverage waiver _was_ recommended in the V2 review (the one that already happened)

Reply:
Ohh, really? Hmm... I suppose we need to firm up our guidance on how to deal with situations like this and stick to it.

Reply:
Not a bad idea, though FWIW the review was a >6 months ago.  These opinions have also been known to change.